src/devtools/client/debugger/src/actions/sources/loadSourceText.js

```diff
@@ -5,6 +5,7 @@
 //
 
 import { PROMISE } from "ui/setup/redux/middleware/promise";
+import { parser } from "devtools/client/debugger/src/utils/bootstrap";
```

Comment:
is this still needed?

Reply:
Yes, `parser` is used on line 46 - this is one of the couple files that referenced `parser` as a thunk extra arg, and I've eliminated that by switching to a direct import.